service/history/historyEngine_test.go

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2017 Uber Technologies, Inc.
+// Copyright (c) 2017 - 2020 Uber Technologies, Inc.
```

Comment:
Let's follow the format in the license file: `Copyright (c) 2017-2020 Uber Technologies Inc.`

Reply:
updated